freetextresponse/tests.py

```diff
@@ -260,6 +261,23 @@ def test_compute_score(self, credit):
             {'value': credit.value, 'max_value': Credit.full.value},
         )
 
+    @ddt.data(Credit.zero, Credit.half, Credit.full)
+    def test_compute_score_error(self, credit):
+        # pylint: disable=protected-access
+        """
+        Tests _compute_score's error handling
```

Comment:
Same here too, let's use IntegrityError in the first line of the comment.  And be specific in the other parts of the comment as well. 

Does this test actually test that all exceptions are handle?  Or is it just the one IntegrityError?